### ISSUE TITLE
Conflict with other extensions

### ISSUE THREAD

**Issue by lawnzapper** [extend-chrome/storage#14]
### Describe the bug

I'm building a Chrome Extension with manifest v3, bundled with ejected CRA and custom webpack config.
Whenever I install it, both unpacked or from the store, some other extensions as Adblock and Http Everywhere crash, and my extension fails to initialize. I found out that my issue was related to @extend-chrome/storage import, so I removed it and used chrome.storage api with no external dependencies.
I'm sorry but I have not time to create a minimal reproduction right now, since I'm full making an official release of the extension I'm working on.
My thoughts are, maybe it's related to the way manifest v3 works now with service workers, or something creates conflict with the use of rxjs and/or chrome-promise.
Don' t worry about to or not to accept the issue since it's not well compiled, I'm just writing to you for the sake of your own library that was easy to use for me. Bye

### Please complete the following information:

- Library Version:
- Operating System:
- Browser:
- Node Version:

**Comment by jacksteamdev** [comment]
@lawnzapper Thanks for the heads up!

That's really weird stuff, definitely a bug in Chrome, since one extension should never be able to crash other extensions. I'll take a look soon.

Question: Are you using the `unlimitedStorage` permission?

**Comment by lawnzapper** [comment]
> Are you using the `unlimitedStorage` permission?

No I'm not, just `storage`.
Were you able to reproduce the bug?

**Comment by jacksteamdev** [comment]
@lawnzapper No, I wasn't, but if you put a link for a reproduction repo, I can look at it on Tuesday.
